rd-net/Directory.Build.props

```diff
@@ -15,6 +15,14 @@
     <Nullable Condition="'$(TargetFramework)' == 'net35'">disable</Nullable>
     <Nullable Condition="'$(Nullable)' == ''">enable</Nullable>
 
+    <!-- A rather hacky way to fix compilation on ARM64
+      Error	MSB4184	The expression "[MSBuild]::DoesTaskHostExist(CLR2, CurrentArchitecture)" cannot be evaluated.
+      MSB0001: Internal MSBuild Error: ARM64 CLR2 task hosts are not supported.
+      C:\Program Files\Microsoft Visual Studio\2022\Preview\MSBuild\Current\Bin\arm64\Microsoft.Common.CurrentVersion.targets 3304
+    -->
+    <GenerateResourceMSBuildRuntime Condition="'$(PROCESSOR_ARCHITECTURE)' == 'ARM64'">x64</GenerateResourceMSBuildRuntime>
```

Comment:
Do we have an issue or error report or something to track? I'd like us to know when it's allowed to remove this workaround.

Reply:
Thanks, I've rechecked and actually it isn't actual anymore. It was fixed in `Microsoft Visual Studio Professional 2022 (ARM 64-bit) - Current Version 17.4.1`, which was released after I started this branch. 

I've force-pushed and removed this hack.